src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -770,19 +769,20 @@ class DocBaseViewer extends BaseViewer {
      */
     initPdfViewerClass(PdfViewerClass) {
         const { file, location } = this.options;
+        const { AnnotationMode: PDFAnnotationMode = {} } = this.pdfjsLib;
```

Comment:
Do these changes (this line and `renderInteractiveForms`) need to both exist while the legacy and latest pdfjs versions are possible?

Reply:
I need to re-add the `renderInteractiveForms: false` line, since it's used by the legacy version. I don't think they conflict with each other, though.

Reply:
@karelee7 Did we test that PDF forms are disabled in modern/ie11 browsers?

Reply:
Yup! I just tested/am in the process of adding `renderInteractiveForms: false` into my commit